xarray/backends/api.py

```diff
@@ -6,6 +6,15 @@
 from glob import glob
 from io import BytesIO
 from numbers import Number
+try:
+    try:
+        from pathlib import Path
+    except ImportError as e:
+        from pathlib2 import Path
+    path_type = (Path, )
+except ImportError as e:
+    path_type = ()
```

Comment:
this looks right to me but we should put it in `pycompat.py`

Reply:
Done.